api/print.php

```diff
@@ -51,9 +55,9 @@
         $source = imagecreatefromjpeg($filename_source);
         $code = imagecreatefrompng($filename_codes);
 
-        if ($config['print_frame'] == true) {
+        if ($config['print_frame'] == true && !($config['take_frame'])) {
             $print = imagecreatefromjpeg($filename_source);
-            $rahmen = @imagecreatefrompng('../resources/img/frames/frame.png');
+            $rahmen = @imagecreatefrompng($print_frame);
```

Comment:
Is there a reason you suppress errors? It makes it harder to debug an issue.

Reply:
Copy paste from print.php

Reply:
It was there already